combat-trainer.lic

```diff
@@ -391,9 +394,19 @@ class LootProcess
     end
   end
 
-  def should_perform_ritual(prepare_consume)
-    !prepare_consume &&
-      ((@necro_train_first_aid && DRSkill.getxp('First Aid') < 32) || DRSkill.getxp('Thanatology') < 32)
+  def can_perform_training_ritual?(game_state)
```

Comment:
I broke the old method, `should_perform_ritual` into two methods here. Training rituals are rituals which are done for training purposes, like harvest and dissect. They do nothing but train the thanatology skill.

Reply:
This looks a bit like PR #1629. I wonder if that logic would help here, and you could prioritize healing > zombie > training when picking a new ritual each time.

Reply:
Ill check, however the prioritization of healing > zombie is already in place, on line 419 and 424

Reply:
I'm not seeing anything to borrow from that PR, let me know if i overlooked anything